src/Codeception/Module/REST.php

```diff
@@ -577,7 +577,7 @@ protected function execute($method, $url, $parameters = [], $files = [])
         // allow full url to be requested
         if (strpos($url, '://') === false) {
             $url = $this->config['url'] . $url;
-            if ($this->config['url'] && strpos($url, '://') === false && $this->config['url'][0] !== '/') {
+            if ($this->config['url'] && strpos($url, '://') === false && strpos($url, '/') !== 0 && $this->config['url'][0] !== '/') {
```

Comment:
Are you sure that this fix was correct?
Previous line overrides `$url` so the first byte of `$url` and `$this->config['url']` is the same.

Reply:
I'll check and redo that if required.

Reply:
https://github.com/Codeception/Codeception/pull/5511